app/controllers/admin/applications_controller.rb

```diff
@@ -0,0 +1,27 @@
+class Admin::ApplicationsController < ApplicationController
+  
+  def approve_pet
```

Comment:
A more RESTful approach here would be to just have one `update` action in this controller and pass `approved` or `denied` as query parameters (will add a note in the view about how you can do this). These two actions are nearly identical so we don't want to duplicate logic, but make a more flexible update action. Also, per REST, we want to name our controller actions only using CRUD actions